### ISSUE TITLE
Naming conventions

### ISSUE THREAD

**Issue by ddsferreira** [rubyjedi/soap4r#15]
Hi Lawrence,

Why not speak with rubygems and ask for the ownership of the soap4r gem itself?
Wouldn't this gather the community around your efforts.
It seems the other forks you mention are pretty much dead as well.
It makes sense in my point of view to make soap4r-ng the standard ruby soap gem.
So it would be like the new major version of soap4r.

If anything you can put the proposal to the ruby talk list.
I'm sure people interested in good soap libraries will back you up.

Think about it.

Regards,

Daniel

**Comment by rubyjedi** [comment]
I agree, there is still a great deal of confusion when it comes to ownership of soap4r.

I have been reluctant in the past to contact NaHi or RubyGems about taking over as the "official" soap4r RubyGem source, as I didn't want to steal the momentum from anyone else actively maintaining their own fork.

Much of what I'm doing is to keep a legacy App I wrote a decade ago alive, as it's still actively used and generating a revenue stream. Whereas my focus is on ensuring old stuff still works, others may be focused on enhancements or workarounds to handle the more gnarly WSDLs out there.

I will definitely keep this in mind, though, (I agree, Soap4R should have a single banner going forward). Perhaps it's time to contact some of the others with Soap4R forks to see if we can unify and consolidate.
